api/external/envoy/api/v2/core/base.proto

```diff
@@ -82,28 +84,70 @@ message Locality {
   string sub_zone = 3;
 }
 
+
```

Comment:
The section of changes in this file were copied in API updates that the previous file did not include (but Gloo needed)